app/assets/javascripts/components/edit_modal.es6.jsx

```diff
@@ -0,0 +1,52 @@
+class EditModal extends Component {
+
+  static get propTypes() {
+    return {
+      student: React.PropTypes.object.isRequired,
```

Comment:
In the future if we abstract this out we might not always pass in a `student` prop (nothing needs to change just to keep in mind) - another option would be to make this a `StudentEditModal`, we'll think more about it later. :+1: